dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -61,6 +61,21 @@ def __init__(self, numerical_aggregation, non_numerical_aggregation=None):
             non_numerical_aggregation or numerical_aggregation
         )
 
+    def get_dtype_relevant_aggregation(self, dtype):
+        numeric_agg = self.numerical_aggregation
```

Comment:
Not a strong preference but might be worth renaming `numeric_aggregation` `numeric_agg` to something that specifies that these aggregations are not specific to `numerical` types only. 
I don't have a good suggestion for alternative names, but I think of them as `supported` or `native` aggregations.

Reply:
Changed it to `native` and `custom` with comit https://github.com/dask-contrib/dask-sql/pull/236/commits/86b3132f28e366524fb33cd499bc0b223307d04d . Let me know if you would prefer another name,